### PR TITLE
[DROOLS-3805] All line separators in RHS in spreadsheet are escaped

### PR DESCRIPTION
**Pull request by tkobayas**
- Unit test

### CODE REVIEW

File: drools-decisiontables/src/test/java/org/drools/decisiontable/LineBreakXLSTest.java [hunk 1]
```diff
@@ -41,4 +42,30 @@ public void makeSureAdditionalCodeLineEndsAreNotAdded() {
...
+        KieSession ksession = kbase.newKieSession();
+
+        Person john = new Person("John");
+        john.setAge(20);
+        ksession.insert(john);
+        ksession.fireAllRules();
+
+        ksession.dispose();
+
+        assertEquals(30, john.getAge());
```
Comment (jomarko) [inline drools-decisiontables/src/test/java/org/drools/decisiontable/LineBreakXLSTest.java:71]: As the issue is about multiple lines in actions, and you set two parameters in the action, each on a separate line, could you please extend this test like this?

```java
        Person john = new Person("John");
        john.setAge(20);
        john.setIsAlive(true);
        ksession.insert(john);
        ksession.fireAllRules();

        ksession.dispose();

        assertEquals(30, john.getAge());
        assertFalse(john.isAlive());
```

**Review by jomarko — COMMENTED** [review]
@tkobayas thank you, this is always very helpful to have reproducer test for the reported issue, good job! Just one minor comment from my site.

**Comment by tkobayas** [comment]
@jomarko Yes, updated. Thank you for the suggestion!

**Comment by jomarko** [comment]
@tkobayas thank you! We have two options now, we can either keeps this opened until the fix for the issuea apperas, or we can add ignore annotation, merge, and unignore as part of the fix, what is your preference?

**Comment by tkobayas** [comment]
@jomarko Either is fine to me. Please go ahead the way which you prefer :)

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
ok to test

**Comment by jomarko** [comment]
@kiegroup/gatekeepers please merge

**Review by jomarko — APPROVED** [review]

**Pull request merged**